src/LibVLCSharp.Avalonia.Unofficial/LibVLCSharp.Avalonia.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
This sln is not needed as well, since we will re-use the current avalonia project.